doc/sphinx/proof-engine/tactics.rst

```diff
@@ -1676,11 +1639,40 @@ Controlling the proof flow
    .. exn:: @ident is used in conclusion.
       :undocumented:
 
-.. tacn:: generalize @term
-   :name: generalize
+   .. tacn:: specialize_eqs @ident
+      :undocumented:
+
+.. tacn:: hresolve_core ( @ident := @one_term ) {? at @nat_or_var } in @one_term
+   :undocumented:
+
+.. tacn:: hget_evar @nat_or_var
+   :undocumented:
```

Comment:
I did some commit archeology and found not only the commit that introduced these tactics 028cbb32785b559c637f77864ce5172e0255d0d0, but also https://sf.snu.ac.kr/gil.hur/Hpattern/readme.txt, which explains what they are for. So they seem to be some internal tactics for a library implementing a generalization of `pattern` that is distributed on a website. We should investigate whether this library is actively maintained, still useful, and justifies keeping these tactics / whether it should be integrated into Coq or Coq-community and distributed more widely. In the meantime, I propose to just move these two tactics close to `pattern` and keep this info in a comment.